# goal definition

string frame

---

# result definition

string[] obj_names

---

# feedback
